### ISSUE TITLE
Seer spacebar function stops working

### ISSUE THREAD

**Issue by serfacter** [ccseer/Seer#206]
Just downloaded Seer on windows 10. Spacebar preview function works for a few folders, then suddenly stops working and I can't even right click on it in the windows taskbar to open the Seer menu. I need to go to task manager, quit seer then reopen it. It starts working again for a few folders then stops and the same thing repeats. This doesn't happen if I use a combination of hotkeys e.g. Shift+ Spacebar or Ctrl+spacebar, but this is less convenient.

EDIT: just stopped working with the hotkey combinations too.
EDIT 2: Turns out it only stops working when I try to use it on my external micro SD (where I'm previewing WAV audio files), and doesn't happen when previewing my main computer files.... Unfortunately, it's on this micro SD card that I need to use it the most. 

Any help is appreciated! Cheers

**Comment by ccseer** [comment]
What version?

**Comment by serfacter** [comment]
Hey,

Seer version 0.8.1 

Just did some more testing: The SD card is fine, Seer only stops working every time I try to preview one particular WAV file (also happened when I moved it to my desktop and tried to preview it there). All other WAV files are fine so I can only assume there's an issue with that particular file

If you want more info on this file to look into it further, let me know. You'll have to tell me how to find the extra info as I just looked at the file properties and they're no different to the other WAV files that preview fine (it's not a large file, same bitrate... etc)

Cheers!
